src/include/distributed/multi_router_executor.h

```diff
@@ -9,16 +9,26 @@
 #ifndef MULTI_ROUTER_EXECUTOR_H_
 #define MULTI_ROUTER_EXECUTOR_H_
 
+#include "libpq-fe.h"
```

Comment:
Ack. Fixing. Was curious because e.g. `include-what-you-use` has some functionality to suggest forward-declaration and I haven't been using it for some time but wondered if it might have made the same suggestion.